app/models/application_search_csv.rb

```diff
@@ -25,7 +25,7 @@ def file_name
   private
 
   def headers
-    %i[reference received_at reviewed_at caseworker status].map do |key|
+    %i[reference received_on reviewed_on caseworker status].map do |key|
```

Comment:
interesting - do we need to change the CSV download for the addition of a closed cases page? Thinking as it applies to the search atm